src/python/director/startup.py

```diff
@@ -1086,12 +1086,6 @@ def stop(self):
     #ikPlanner.fixedBaseArm = True
     #showImageOverlay()
 
-
-if 'exo' in drcargs.args():
```

Comment:
I believe this also requires a change to https://github.com/openhumanoids/director/blob/fix-ik-matlab-communication/src/python/director/drcargs.py#L77 or replace the functionality here to set ``self.ikPlanner.planningMode = 'exotica'`` if exo is in drcargs.args()